cmd/loom/gateway/general_cmds.go

```diff
@@ -657,6 +661,76 @@ func newSetWithdrawFeeCommand() *cobra.Command {
 	return cmd
 }
 
+func newSetMaxWithdrawLimitCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "set-max-withdrawal-limit <total-amount-limit> <per-account-amount-limit> [gateway]",
```

Comment:
It should be possible to change each limit individually:
```bash
set-withdrawal-limit [gateway] --total-limit <amount> --account-limit <amount>
```
So if the user specifies only `--total` then only the total limit should change.